name.tf

```diff
@@ -0,0 +1,13 @@
+resource "random_string" "resource_suffix" {
+  length  = 5
+  lower   = true
+  upper   = false
+  number  = false
+  special = false
+}
+
+locals {
+  service_name  = var.service_name != "" ? var.service_name : "${data.ns_workspace.this.block_ref}"
```

Comment:
We should drop the `service_name` var and always use `block_name` (not `block_ref`)

Reply:
Removed input var `service_name` and replace references with **block_name**